.github/CONTRIBUTING.md

```diff
@@ -225,45 +254,60 @@ export {
 
 ### Unit Testing
 
-Our unit tests are written with [Vitest](https://vitest.dev/) and can be found alongside any javascript/vue code files existing in Dialtone Vue. Tests are suffixed with `.test.js` of the corresponding javascript/vue file. ex: `link.vue` is tested by `link.test.js`. All code in Dialtone Vue should have tests written for it. You can see more details on writing tests in the following [documentation](https://vue.dialpad.design/?path=/story/docs-component-driven-development-unit-tests--page).
+Our unit tests are written with [Vitest](https://vitest.dev/) and can be found alongside any javascript/vue code files existing in Dialtone Vue.
+Tests are suffixed with `.test.js` of the corresponding javascript/vue file. ex: `link.vue` is tested by `link.test.js`.
+All code in Dialtone Vue should have tests written for it.
+You can see more details on writing tests in the following [documentation](https://vue.dialpad.design/?path=/story/docs-component-driven-development-unit-tests--page).
 
 ### Dialtone Usage
 
-Dialtone Vue components should utilize the global immutable CSS classes provided by Dialtone whenever possible. It is a requirement of any project using Dialtone Vue to include Dialtone as a dependency. If needed, you can also write custom CSS using Dialtone LESS variables by importing `../css/dialtone.less`.
+Dialtone Vue components should utilize the global immutable CSS classes provided by Dialtone whenever possible.
+It is a requirement of any project using Dialtone Vue to include Dialtone as a dependency.
+If needed, you can also write custom CSS using [Dialtone Tokens CSS variables](https://dialpad.design/tokens/).
 
-Please **do not** use any mixins in Dialtone Vue components.
+If you made a local style change, please ask in the [#dialtone](https://dialpad.slack.com/messages/dialtone/) Slack channel for a release as dialtone-vue styles are now
+[included within dialtone](https://dialpad.design/about/whats-new/posts/2023-8-14.html).
+
+Please **do not** use any mixins or utility classes in Dialtone Vue components.
 
 ### Breaking Changes
 
-Dialtone Vue is a dependency of other projects so you must always be aware of making a breaking change. It is possible to make breaking changes to a component, however if you do so you will need to update / verify all instances of usage of the component in ubervoice/uberconf before updating.
+Dialtone Vue is a dependency of other projects, so you must always be aware of making a breaking change.
+It is possible to make breaking changes to a component, however if you do so you will need to update / verify
+all instances of usage of the component in ubervoice/uberconf before updating.
```

Comment:
Add that we should also communicate the breaking change via a blog post on dialpad.design.